src/cata_tiles.cpp

```diff
@@ -1429,10 +1429,8 @@ void cata_tiles::draw_zones_frame()
 void cata_tiles::draw_footsteps_frame()
 {
     static const std::string footstep_tilestring = "footstep";
-    std::vector<point> markers;
-    g->calculate_footstep_markers(markers);
-    for (std::vector<point>::const_iterator a = markers.begin(); a != markers.end(); ++a) {
-        draw_from_id_string(footstep_tilestring, a->x, a->y, 0, 0);
+    for( const auto &sound : g->sound_markers ) {
```

Comment:
`sound_markers` isn't a member of `game` anymore. This broke the tiles builds -- I've temporarily fixed it (via #11030) by making `sounds::sound_markers` accessible for anyone who includes `sounds.h`, but that's almost certainly the wrong approach. I have no idea what the right way to do it would be, though.